.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```

Comment:
gitignore your .vscode folder as it relates to your vscode not your project

Reply:
also delete it from your git cache to git rid of it here (google how to do this)